modules/gittar/pkg/gc/gc.go

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// This program is free software: you can use, redistribute, and/or modify
+// it under the terms of the GNU Affero General Public License, version 3
+// or later ("AGPL"), as published by the Free Software Foundation.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+package gc
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os/exec"
+	"strings"
+
+	"github.com/erda-project/erda/modules/gittar/pkg/limit_sync_group"
+	"github.com/erda-project/erda/pkg/cron"
+	"github.com/sirupsen/logrus"
+)
+
+// all gittar repository root address
+const repositoryRootAddr = "/repository"
+
+// gc repository concurrent number
+const concurrentNum = 1
+
+// gc repository cron time
+const cronExpression = "0 0 1 * * ?"
```

Comment:
do we need get cron expresssion from config? Maybe there were some scenes need different gc cycle